vuepress/docs/next/tutorials/ecr/deploy-components-without-bundle.md

```diff
@@ -1,46 +1,54 @@
 ---
 redirectFrom: /next/tutorials/ecr/tutorials/ecr-deploy-use-plugin-and-mfe-without-bundle.html
 ---
-# Use Blueprint Generated Plugin and Micro Frontends Without a Bundle
+# Apply Entando Blueprint Without a Bundle
 
 ## Overview
 
-In this tutorial you will learn how to use the plugin and microfrontend generated using the Entando Blueprint in a running cluster without the need to include the components in a bundle.
-
-**If you haven't already generated a plugin and micro frontends with the Component Generator go here first [Generate a Plugin](../backend-developers/generate-microservices-and-micro-frontends.md)**
+In this tutorial you will learn to apply an Entando Blueprint to a plugin and Micro Frontend generated with the Entando Component Generator in a running cluster, without the need for a bundle.
```

Comment:
learn how to apply --added how
i've seen (M)micro (F)frontend every which way... capitalization rule going forward?
this sentence is a little long and needs to be reread to clarify... suggest breaking it up, reorganizing, or adding "that" (that are generated)

Reply:
reworded. Generally it seems to be capitalized, I've been doing it that way but don't catch it sometimes.